packages/authorization-process/test/getClientKeys.test.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  getMockClient,
+  getMockKey,
+} from "pagopa-interop-commons-test/src/testUtils.js";
+import {
+  Client,
+  Key,
+  UserId,
+  generateId,
+  unsafeBrandId,
+} from "pagopa-interop-models";
+import { describe, expect, it } from "vitest";
+import { genericLogger } from "pagopa-interop-commons";
+import {
+  clientNotFound,
+  organizationNotAllowedOnClient,
+} from "../src/model/domain/errors.js";
+import { addOneClient, authorizationService } from "./utils.js";
+
+describe("getClientKeys", async () => {
+  const consumerId = generateId();
+  const mockKey = getMockKey();
+  const mockClient: Client = {
+    ...getMockClient(),
+    keys: [mockKey],
+    consumerId: unsafeBrandId(consumerId),
+  };
+
+  it("should get the keys in the specified client", async () => {
+    await addOneClient(mockClient);
```

Comment:
may you add a couple of keys with different users and verify that they are returned?